src/ogt_voxel_meshify.h

```diff
@@ -688,6 +697,12 @@ ogt_mesh* ogt_mesh_from_paletted_voxels_simple(
                     mesh->vertex_count += 4;
                     mesh->index_count  += 6;						
                 }
+
+                uint32_t count = mesh->vertex_count - start_vert_count;
+                if (NULL != ctx->emit_verts_cb)
+                {
```

Comment:
I would recommend renaming emit_verts_cb -> emit_verts_func and cb_ctx -> emit_verts_func_user_data for consistency.